### DISCUSSION TITLE
Creazione pulsante virtuale

### DISCUSSION THREAD

**Discussion by fabplus75** [indomus/forum#1691] [GENERALE - Discussioni generiche]
buongiorno, mi servirebbe creare un pulsante virtuale che, quando lo vado a premere, a sua volta mi prema in sequenza ad una distanza di un secondo l'uno dall'altro due pulsanti . Per favore mi aiutate. Grazie.

**Comment by Scialla** [comment]
https://www.home-assistant.io/integrations/input_boolean/

Ti crei il pulsante virtuale e poi fai una automazione, come trigger lo stato dell'input boolean che va a on, come action quello che ti serve e, alla fine, riporti l'input boolean a off

basico proprio..

**Comment by 50494554524F** [comment]
puoi usare button, input button con relativi script o, più semplice da mantenere, switch template con lo stato sempre su false in modo che funzioni da push button
